contract/request.go

```diff
@@ -25,12 +25,17 @@ func (operator *Operator) Register(code []byte) {
 func (operator *Operator) sendTx(toAddr string, txType int, payload []byte) {
 	hash := crypto.Keccak256Hash(payload)
 
+	sig, err := crypto.NewSignature(operator.privKey, hash)
+	if err != nil {
+		log.Error(err.Error())
```

Comment:
if something wrong, do we need to return?

Reply:
just keep what it is before, maybe we will refine code later

Reply:
change to panic